### DISCUSSION TITLE
Is it possible to log every call to t() to detect unused keys?

### DISCUSSION THREAD

**Discussion by hlassiege** [i18next/i18next#1816] [Q&A] [answered]
Hi,

I'm using i18next with i18next-vue. 
I'm wondering if there is a configuration somewhere to plug my own callback before every access to $t()
My goal would be to keep somewhere all keys used and then to audit from time to time if I have some unused keys in my application.

**Comment by jamuhl** [comment, marked as answer]
For our translation management we created https://github.com/locize/locize-lastused and intercept basically accessing keys on the resource store: https://github.com/locize/locize-lastused/blob/master/lib/index.js#L58

=> With our locize.com (from the core maintainers of i18next) you get that feature plus a nice UI to bulk delete unused resources.

**Comment by hlassiege** [comment]
thanks a lot. This is a very good starting point.
